openapi/components/schemas/Plans/Plan.yaml

```
@@ -12,3 +13,33 @@ allOf:
         minItems: 1
         items:
           $ref: ../Links/SelfLink.yaml
+      invoiceIssueTimeShift:
+        type: string
+        description: >
+          The calculation instruction of billing time.
+
+
+          This is used in conjunction with the **service period anchor** to calculate
+          the time the invoice is issued.
+
+
+          Time period between the invoice issue time shift and the invoice due time shift in **ISO-8601** format.
+        nullable: true
+        default: '-P00Y00M00DT00H00M00S'
+        example: '-P7D'
+      invoiceDueTimeShift:
+        type: string
+        description: >
+          The calculation instruction of due time.
+
+
+          This is used in conjunction with the issue time to calculate due time of invoice.
+
+
+          The chronology of due time shift is always **after**.
+
+
+          Time period between the invoice issue time shift and the invoice due time shift in **ISO-8601** format.
```

```suggestion
          Time period between the invoice issue time shift and the invoice due time shift in **ISO-8601** format.
          Period must be positive.
```
The term `chronology` seems not relevant to this value anymore

---

Updated as Adam requested, thank you